### ISSUE TITLE
Alarm volume slider should lose the hint and play the alarm sound while changing volume

### ISSUE THREAD

**Issue by dark-eye** [ubports/clock-app#11]
**Duplicate of LP issue at : https://bugs.launchpad.net/ubuntu-clock-app/+bug/1362078**

The alarm volume slider in settings displays a hint when changing volume and says just a number between 0 and 100.

This number isn't really relevant to the user, what is, instead, is the actual alarm volume at the selected slider position.

--- Solution ---
- The alarm sound should be played while you're touching the slider (i.e. the slider is focused).
- When the slider loses focus or the user leaves the screen, the alarm volume fades out

Additional comment:
To have common User Experience across all application, it will be great to implement volume preview also for "System Settings -> Sound -> Ringer":
https://bugs.launchpad.net/ubuntu-ux/+bug/1479374

**Comment by Flohack74** [comment]
BTW I am really missing individual sound settings also for other things. The master volume control should be sensitive to whats going on, i.e. when a call comes in, I want to moderate the level of the ringer sound for the phone, but not also change my playback level for media. Every time when a sound is being played the phone could be context-sensitive. But I know, this is a big dream xD

**Comment by dark-eye** [comment]
Yeah thats my dream too :) Also integrate it with a phone state detection engine that can detect whats  the state of the phone (in pocket / laying on table / in car dock / etc.. ). 
but i think it`s more of a core thing so it will take me some time to get to it ;)

**Comment by Flohack74** [comment]
Please make enhancement tickets for that, in the ubports/ubuntu-touch project ;)
